env_batch.xml

```diff
@@ -150,7 +150,7 @@
       <valid_values/>
       <desc>Override the batch submit command this job. Do not include executable or dependencies</desc>
     </entry>
-    <entry id="JOB_WALLCLOCK_TIME" value="1:00:00">
+    <entry id="JOB_WALLCLOCK_TIME" value="6:00:00">
```

Comment:
Might need to change more:

0[102] cheyenne2:/<3>Exp/f.e21.FHIST_BGC.f09_025.CAM6assim.011 > ./xmlquery --partial WALL

Results in group case.run
	JOB_WALLCLOCK_TIME: 12:00:00
	USER_REQUESTED_WALLTIME: 12:00

Results in group case.st_archive
	JOB_WALLCLOCK_TIME: 6:00:00
	USER_REQUESTED_WALLTIME: 1:00

Reply:
Thanks.  I pushed the fix.